src/schema.py

```diff
@@ -100,16 +100,16 @@ def resolve_account_info(self, info, session_id=None):
     account_info['brbs'] = '0.00'
     account_info['city_bucks'] = '0.00'
     account_info['laundry'] = '0.00'
+    account_info['swipes'] = ''
     for acct in accounts:
       if acct['accountDisplayName'] == ACCOUNT_NAMES['citybucks']:
         account_info['city_bucks'] = str("{0:.2f}".format(round(acct['balance'], 2)))
       elif acct['accountDisplayName'] == ACCOUNT_NAMES['laundry']:
         account_info['laundry'] = str("{0:.2f}".format(round(acct['balance'], 2)))
       elif ACCOUNT_NAMES['brbs'] in acct['accountDisplayName']:
         account_info['brbs'] = str("{0:.2f}".format(round(acct['balance'], 2)))
-      # Need more research to implement swipes:
-      # Each plan has a different accountDisplayName
-      account_info['swipes'] = ''
+      elif any(meal_swipe_name in acct['accountDisplayName'] for meal_swipe_name in ['Bear Basic', 'Bear Choice', 'Bear Traditional']):
```

Comment:
why not define `['Bear Basic', 'Bear Choice', 'Bear Traditional']` in a constant up top?

also, i am not a fan of this nested one-line for loop, but we can verify with @meganle

Reply:
Not sure. I don't really do python backend, was just trying to fix a feature

Reply:
yeah, i think it's best if the array is a constant.

also is there any reason that you use `any(...)`?

Reply:
stackoverflow